src/components/info-list-item.tsx

```diff
@@ -0,0 +1,72 @@
+import { FC, ReactNode } from 'react';
+import classNames from 'classnames';
+
+import '../styles/components/info-list-item.scss';
+
+type Props = {
+  /**
+   * Title
+   */
+  title: string;
+
+  /**
+   * Textual content
+   */
+  content: string;
+
+  /**
+   * Same as 'content', but can accept HTML elements
+   */
+  children: string | ReactNode;
+
+  /**
+   * Make this item visually stand-out
+   */
+  highlight?: boolean;
+
+  /**
+   * Compact style
+   */
+  compact?: boolean;
+
+  /**
+   * Hide the title
+   */
+  hideTitle?: boolean;
+
+  /**
+   * Attempts to keep the element horizontally aligned with its siblings
+   */
+  inline?: boolean;
+};
+
+const InfoListItem: FC<Props> = ({
+  title,
+  content,
+  children,
+  highlight,
+  compact,
+  hideTitle,
+  inline,
```

Comment:
Please add `className` and `...props` as per other components in Franklin so that we can pass classes and other props from higher to the underlying `div`.
Not sure if that might actually help you with passing classes from the `InfoList` components and solve some of the inconsistencies you noticed you had to do when extracting this component as standalone

Reply:
done!